connectors/http-json/src/main/java/io/camunda/connector/http/auth/CustomAuthentication.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.http.auth;
+
+import com.google.api.client.http.HttpHeaders;
+import io.camunda.connector.api.annotation.Secret;
+import io.camunda.connector.http.model.HttpJsonRequest;
+import java.util.Map;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+public class CustomAuthentication extends Authentication {
+
+  @NotNull @Valid private HttpJsonRequest request;
+
+  @Secret private Map<String, String> outputBodyMap;
```

Comment:
I think this should be `outputBody` instead of `outputBodyMap`. In the element template I was able to find outputBody but not outputBodyMap binding, and when I tested this was always null.